processor/routingprocessor/README.md

```diff
@@ -51,14 +51,14 @@ exporters:
     endpoint: localhost:24250
 ```
 
-### Tech Preview: OpenTelemetry Transformation Language expressions as routing conditions
+### Tech Preview: OpenTelemetry Transformation Language statements as routing conditions
 
-Alternatively, it is possible to use subset of the [OpenTelemetry Transformation Language (OTTL)](https://github.com/open-telemetry/opentelemetry-collector/blob/main/docs/processing.md#telemetry-query-language) expressions as routing conditions.
+Alternatively, it is possible to use subset of the [OpenTelemetry Transformation Language (OTTL)](https://github.com/open-telemetry/opentelemetry-collector/blob/main/docs/processing.md#telemetry-query-language) statements as routing conditions.
```

Comment:
Unrelated to this PR, but I'd recommend linking to the OTTL readme instead of the processing doc which is more theoretical.

Reply:
done